### PR TITLE
Try to clarify the documentation for -x command

### PR DESCRIPTION
**Pull request by stephane-archer**
see issue #712

### CODE REVIEW

**Review by tmccombs — APPROVED** [review]

**Comment by stephane-archer** [comment]
What do you think about this change? Do I need to do something to help the review or merge process?

**Comment by sharkdp** [comment]
Looks good - thank you very much. It would be great if we could also modify the `--help` text (`src/app.rs`).

File: doc/fd.1 [hunk 1]
```diff
@@ -261,7 +261,7 @@ Provide paths to search as an alternative to the positional \fIpath\fR argument.
 .BI "\-x, \-\-exec " command
 Execute
 .I command
-for each search result. The following placeholders are substituted by a path derived from the current search result:
+for each search result in parallel (see --thread 1 for sequential order). The following placeholders are substituted by a path derived from the current search result:
```
Comment (sharkdp) [inline doc/fd.1:264]: It should be `--threads 1` instead of `--thread 1`. Maybe:

`(use --threads=1 for sequential command execution)`

Comment (stephane-archer) [reply doc/fd.1]: I think you are right

Comment (stephane-archer) [reply doc/fd.1]: I think you are right

**Comment by stephane-archer** [comment]
Do you have any feedback?

**Comment by sharkdp** [comment]
Looks good - thank you very much!

**Comment by stephane-archer** [comment]
@sharkdp no problem, I'm glad if I help. Please feel free to do what feels right for you.

**Pull request merged**